libs/lein-template/deps.edn

```diff
@@ -1,3 +1,4 @@
-{:paths   ["src" "resources"]
+{:paths   ["src"]
  :deps    {selmer/selmer                   {:mvn/version "1.12.50"}
-           com.github.seancorfield/clj-new {:mvn/version "1.2.381"}}}
+           com.github.seancorfield/clj-new {:mvn/version "1.2.381"}
+           io.github.kit-clj/deps-template {:mvn/version "0.1.25"}}}
```

Comment:
At first I tried using `{:local/root "../deps-template"}` but I ran into issues with the `build.clj` tasks when I tried to do that.

Reply:
adding deps template here seems fine